src/modules/dashboard/hooks/useAvatarDisplayCounter.ts

```diff
@@ -0,0 +1,37 @@
+import Maybe from 'graphql/tsutils/Maybe';
```

Comment:
I think we should move all hooks in the `dashboard` module inside `utils/hooks`.

They really have no place being here.

But other then that, very nice of you for building this hook.